client/joinus_client/src/components/Events/EventCard.js

```diff
@@ -193,8 +250,206 @@ export default function EventCard(props) {
             />
           </CardActions>
         )}
-
       </Card>
+      <Modal
+        open={open}
+        onClose={(e) => setOpen(false)}
+        aria-labelledby="modal-modal-title"
+        aria-describedby="modal-modal-description"
+        sx={StyledModal}
+      > 
+        <Box 
+        width={500} 
+        height={700} 
+        bgcolor="white" 
+        p={3} 
+        borderRadius={3}
+        sx={FormBox}
+        onSubmit={(event) => {
+          event.preventDefault();
+          const sendDataObj = {
+            eventName,
+            eventAddress,
+            eventImage,
+            eventDescription,
+            eventSizeLimit: 2,
+            eventOwnerId: user.id, // grab owner_id from cookies .. just user user.id?
+            eventCategory,
+            lat: 51.0233064354121, // will eventually need to generate these values from address
```

Comment:
these values should automatically generate from the address...

Reply:
Refer to: `// will eventually need to generate these values from address`